src/Components/App/App.js

```diff
@@ -8,13 +8,20 @@ import ReflectionLibrary from '../ReflectionLibrary/ReflectionLibrary.js';
 import FeelingRoom from '../FeelingRoom/FeelingRoom.js';
 import BreathingRoom from '../BreathingRoom/BreathingRoom.js';
 import openLogo from '../../Assets/openLogo.svg';
+import {getQuote} from '../../apiCalls.js'
 
 class App extends Component {
   constructor(props){
     super(props);
     this.state = {
+      quote: '',
+      mood: ''
     }
   }
+  async componentDidMount() {
+    let quote = await getQuote()
+    this.setState({quote: quote.content})
```

Comment:
That original quote api didn't match the app very well. This one is much better!